cioc-rsd/public/cioc-rsd-public.php

```diff
@@ -121,11 +121,13 @@ public function enqueue_styles() {
 		}
 		if (!$no_bs) {
 			wp_enqueue_style( 'bs-glyphicons', 'https://d3byedob0d0n2o.cloudfront.net/bootstrap-3.3.5/glyphicons.css');
+			wp_enqueue_style( 'bs-modal',  $this->parent->plugin_main_dir . 'css/bootstrap-modal.min.css', array(), $this->version);
 		}
 	}
 
 	public function enqueue_scripts() {
 		wp_enqueue_script( $this->plugin_name, plugin_dir_url( __FILE__ ) . 'js/cioc-rsd-public.js', array( 'jquery' ), $this->version, TRUE );
+		wp_enqueue_script( 'bs-modal-script', plugin_dir_url( __FILE__ ) . 'js/bootstrap-modal.min.js', array( 'jquery' ), $this->version, TRUE );
```

Comment:
Should this have the "no bs" if statement - since a bootstrap-ready site will already have this?